testing/scenario_app/ios/Scenarios/Scenarios/AppDelegate.m

```diff
@@ -9,6 +9,18 @@
 #import "ScreenBeforeFlutter.h"
 #import "TextPlatformView.h"
 
+// A UIViewController that sets YES for its preferedStatusBarHidden property.
+@interface NoStatusBarViewController : UIViewController
+
+@end
+
+@implementation NoStatusBarViewController
+- (BOOL)prefersStatusBarHidden {
+  return YES;
+}
+@end
```

Comment:
How is this changes to this file relevant?

Reply:
When testing with XCode 15, I found that the statusbar are showing for certain tests. This is not relevant to this particular fix. I can move it to a different PR.

Reply:
Done